nibabel/gifti/parse_gifti_fast.py

```diff
@@ -168,13 +168,14 @@ def StartElementHandler(self, name, attrs):
                 self.da.ind_ord = array_index_order_codes.code[
                     attrs["ArrayIndexingOrder"]]
             if "Dimensionality" in attrs:
-                self.da.num_dim = int(attrs["Dimensionality"])
-            for i in range(self.da.num_dim):
+                self.num_dim = int(attrs["Dimensionality"])
```

Comment:
Cool. Updated and pushed. Tests passed locally, but my battery will die before I can review things on my own. Feel free to wait until I'm able to review myself!